### PR TITLE
Add readonly ApplicationConfig to /internal/config

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Feature

## Description

This adds the values from `ApplicationConfig` to `/internal/configs`.

Note: there are quite a few values, 73 in total. So I opted for a slightly less sophisticated approach, basically just using some Ruby methods to iterate over the `ENVied` config. Since these values are read-only, I thought we could get away with this, since you'd have to edit `Envfile` in order to change them, which includes more comments. This was done both to save space on the page, as well as save time in developing this.

## Related Tickets & Documents

#5384 

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

Since the page got quite long, I made sections that can be toggled:

Screen Shot 2020-01-23 at 14 31 09

The `ApplicationConfig` section looks like this:

Screen Shot 2020-01-23 at 14 31 36

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [X] no documentation needed

### CODE REVIEW

File: app/views/internal/configs/show.html.erb [hunk 1]
```diff
@@ -1,238 +1,282 @@
...
+        <table class="table">
+          <thead>
+            <tr>
+              <th>Variable</th>
+              <th>Value</th>
+            </tr>
+          </thead>
+          <tbody>
+            <% ENVied.config.variables.each do |v| %>
+            <% if v.group.in?(ENVied.required_groups) %>
```
Comment (rhymes) [inline app/views/internal/configs/show.html.erb]: Do we need the check?

Comment (citizen428) [reply app/views/internal/configs/show.html.erb]: If we want this to work in development, then yes. `ENVied.variables` will return them all, but the object won't respond to the method until the relevant group has been required. So locally the page crashes with a `NoMethodError`. The other way is checking for `ENVied.responds_to?(v.name)`, but I liked the group check better, as it somewhat documents why the object might not have a certain method available.

**Comment by citizen428** [comment]
@rhymes Thanks for helping the aesthetically challenged 😉 The sections are now expanded by default and I added some spacing between them.

Screen Shot 2020-01-27 at 14 43 46

Screen Shot 2020-01-27 at 14 44 02

File: app/views/internal/configs/show.html.erb [hunk 2]
```diff
@@ -1,238 +1,285 @@
...
+          </tr>
+          <% end %>
+          <% end %>
+        </tbody>
+      </table>
+    </div>
   </div>
-<% end %>
+</div>
+<//div>
```
Comment (Zhao-Andy) [inline app/views/internal/configs/show.html.erb]: Think this should be `</div>` Although it renders fine anyway 🙃

Comment (citizen428) [reply app/views/internal/configs/show.html.erb]: Checking again this whole `div` seems to have snuck in, probably some autocomplete mess up.

BTW, I won't get offended if you fix typos etc. on "my" branches yourself, especially if it means we can get things merged.

**Pull request merged**